File: 4기 코딩테스트_220615/4기 - 2nd 코딩테스트/CountMatchedPrefixWord/CountMatchedPrefixWord.js
/**
 * @param array {number[]}
 * @param p {string}
 * @return {number}
 */
function solution(array, p) {
  let result = 0

  for (let i = 0; i < array.length; i++) {
    const s = array[i]
    if (s.indexOf(p) === 0) {
      result++
    }
  }

  return result
}

export default solution

File: 4기 코딩테스트_220615/4기 - 2nd 코딩테스트/CountMatchedPrefixWord/CountMatchedPrefixWord.py
def solution(array, p):
    """
    :param array: int[]
    :param p: string
    :return: int
    """

    result = 0

    for i in range(len(array)):
        s = array[i]
        if s.find(p) == 0:
            result += 1

    return result
